### ISSUE TITLE
[Web] [Accessibility] [11yMAS][Name, Role and Value] [MAS4.1.2][HighImpact] [Name and Role is not defined for all the Interactive controls in the Navigation region.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab-2017#702]
**User Experience:**
User who rely on AT will not get proper information about the elements, if  role is not defined properly which provides the purpose of control. Also name is not defined for the all interactive controls on the page.

**Test Environment:**
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0
Tools used: Developer tool

**Attachments:**
![mas4 1 2 hamburger menu and left navigation pane](https://user-images.githubusercontent.com/33054846/32546340-c5831bee-c4a4-11e7-9989-c21aa7dc3ed7.png)
![mas4 1 2 commabr icon gallery list](https://user-images.githubusercontent.com/33054846/32546342-c6b828ce-c4a4-11e7-8db4-db647021597a.png)
[ScreenCapture_11-9-2017 10.47.47 AM.zip](https://github.com/OfficeDev/script-lab/files/1456404/ScreenCapture_11-9-2017.10.47.47.AM.zip)

**Repro Steps:**
1.Open the URL: https://script-lab.azureedge.net/#/edit/excel
2. Navigate to Hamburger icon on the top of the page (3 horizontal lines) in scan mode.Verify that the name and role is not defined for the hamburger menu button.
3. Navigate to all the tab items in the left navigation pane in scan mode. Verify that name and role is not defined for the left navigation pane.
4. Navigate to the Command bar expand and collapse icon in scan mode. Verify that name and role is not defined for the command bar expand collapse button.
3. Navigate to the Gallery list in scan mode and verify that name and role is not defined for the gallery list

**Current Behavior:**
1. Name and role is not defined for the Hamburger menu. Screen reader do not read the name and role for the hamburger menu.
2. Name and role is not defined for the left navigation pane. Screen reader reads as “New snippet” (‘New snippets’, ‘My snippets’, ‘Samples ’, ’Import’)
3. Name and role is not defined for the command bar expand collapse button. Screen reader reads it as “Group”.
4. Name and role is not defined for the gallery list . Screen reader reads as ‘Last updated 7 days ago ‘ and press down arrow again then it reads “article Blank Snippet-1”.

**Expected Behavior:**
1. Name should be defined as ‘Hamburger’ and role ‘menu’ for the Hamburger menu.
2. Name should be defined as ‘New Snippets’  and role ‘menu item’ for the left navigation pane.
3. Name and role should be defined as ‘Command bar expand /collapse button’ for the command bar expand collapse button
4. Name and role should be defined for the gallery list as “Blank snippet -1 button , 1 of 1”

**Note:**
1. Name is also not defined for the Message bar Warning icon.
2. Same Issue is repro for the My Snippet and Samples page. Name and role is not defined for all the interactive controls on the page 

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=4.1.2
